en/docs/deploy/change-datasource-consent-management.md

```diff
@@ -46,15 +47,13 @@ Following are the sample configuration for each database type.
         jmx_enable=false
         ```
         
-    2. Executing database scripts. 
+    2.  Execute the database scripts. 
     
-        Navigate to `<IS-HOME>/dbscripts`. Execute the scripts in the following file, against 
+        Navigate to `<IS-HOME>/dbscripts`. Execute the scripts in the `<IS-HOME>/dbscripts/consent/mysql.sql` file against 
```

Comment:
shall we follow the same approach done in https://github.com/wso2/docs-is/pull/2589/files#diff-c5d3eb6d8f18f262d91cdc710adc2812ff390034c7cb7f25fe52ff0e1b5f2d03R59. ? Please check all applicable files.